tests/search/attributesearch/attributesearch.rb

```diff
@@ -56,12 +56,18 @@ def a_r_w_e(word, resnam, explain)
     assert_result("query=fstitle:#{word}", selfdir + "case.#{resnam}.result", nil, nil, 0, explain)
   end
 
-  def test_attributesearch_case
-    deploy_app(SearchApp.new.sd(selfdir+"cased.sd"))
+  # helper method
+  def expect_fail(word, explain)
+    assert_hitcount("query=title:#{word}", 0)
+    assert_hitcount("query=fstitle:#{word}", 0)
+  end
+
+  def test_attributesearch_uncased
+    deploy_app(SearchApp.new.sd(selfdir+"uncased/cased.sd"))
```

Comment:
Consider renaming the sd-file to _test.sd_ as this naming could be misleading.

Reply:
Fixed